shared_model/validators/settings.hpp

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef IROHA_SHARED_MODEL_SETTINGS_HPP
+#define IROHA_SHARED_MODEL_SETTINGS_HPP
+
+#include "interfaces/common_objects/types.hpp"
+
+namespace shared_model {
+
+  namespace validation {
+
+    /**
+     * Structure that holds configurable ledger settings
+     */
+    struct Settings {
+      explicit Settings(size_t max_description_size = 64);
```

Comment:
Maybe move `64` to `getDefaultSettings()` definition?

Reply:
Fixed